crates/cargo-remove/src/cargo/ops/cargo_remove/mod.rs

```diff
@@ -75,6 +79,7 @@ pub fn remove(options: &RemoveOptions<'_>) -> CargoResult<()> {
             .warn("aborting remove due to dry run")?;
     } else {
         manifest.write()?;
+        resolve_ws(options.workspace)?;
```

Comment:
Could you add a test that ensures the section is removed from the lock file?

Reply:
Doh, I forgot to unstash and commit that.